test/colony.js

```diff
@@ -1720,4 +1721,88 @@ contract("Colony", addresses => {
       await checkErrorRevert(colony.claimPayout(taskId, MANAGER_ROLE, token.address, { from: OTHER }));
     });
   });
+
+  describe("when in recovery mode", () => {
+    it("should be able to add and remove recovery roles", async () => {
```

Comment:
Allowing this feels like a bad idea...

Reply:
Mm that's a good point. I can change it so you can't add recovery roles in recovery mode.